src/screens/ChatScreen/components/ChatList.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useRef } from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { ChatListItem } from 'app/screens/ChatScreen/components/ChatListItem';
+import { selectors } from 'app/store';
+import { defaultTheme } from 'app/theme/default';
+
+const styles = StyleSheet.create({
```

Comment:
In our starter kit and in most of our projects we keep styles at the end of the file.
What do you think about doing the same here?

Reply:
I moved styles to bottom in whole project